hub/api/dataset.py

```diff
@@ -130,9 +131,10 @@ def __getitem__(
     def create_tensor(
         self,
         name: str,
-        htype: str = DEFAULT_HTYPE,
-        chunk_size: int = None,
-        dtype: str = None,
+        htype: Optional[str] = DEFAULT_HTYPE,
+        chunk_size: Optional[int] = None,
+        dtype: Optional[str] = None,
+        compression: Optional[str] = DEFAULT_COMPRESSION,
```

Comment:
this should be `None`. because if `htype` is specified, but you put `DEFAULT_COMPRESSION` here, the htype will be overwritten by default always

Reply:
@benchislett is going to add tests for these things in another PR don't worry about that for now